integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -317,6 +318,34 @@ def post(self, request, *args, **kwargs):
                 page_translation_form.instance.page = page_form.save()
             # Save page translation form
             page_translation_form.save()
+
+            # Show a message that the slug was changed if it was not unique
+            if user_slug and user_slug != page_translation_form.cleaned_data["slug"]:
+                other_translation = PageTranslation.objects.filter(
+                    page__region=region, slug=user_slug, language=language
+                ).first()
+                other_translation_link = reverse(
+                    "page_revisions",
+                    kwargs={
+                        "page_id": other_translation.page.id,
+                        "language_slug": language.slug,
+                        "region_slug": region.slug,
+                        "selected_revision": other_translation.version,
+                    },
+                )
+                messages.warning(
+                    request,
+                    _(
+                        "The slug was changed from '%(user_slug)s' to '%(slug)s', because '%(user_slug)s' is already used by <a href='%(link)s' class='underline hover:no-underline'>%(translation)s</a>"
+                    )
+                    % {
+                        "user_slug": user_slug,
+                        "slug": page_translation_form.cleaned_data["slug"],
+                        "link": other_translation_link,
+                        "translation": other_translation,
+                    },
```

Comment:
```suggestion
                        "The slug was changed from '{user_slug}' to '{slug}', because '{user_slug}' is already used by <a href='{link}' class='underline hover:no-underline'>{translation}</a>"
                    ).format(
                        user_slug=user_slug,
                        slug=page_translation_form.cleaned_data["slug"],
                        link=other_translation_link,
                        translation=other_translation,
                    ),
```